renderdoc/driver/gl/wrappers/gl_framebuffer_funcs.cpp

```diff
@@ -1708,12 +1708,36 @@ void WrappedOpenGL::glDrawBuffers(GLsizei n, const GLenum *bufs)
   }
 }
 
+template <typename SerialiserType>
+bool WrappedOpenGL::Serialise_glInvalidateNamedFramebufferData(SerialiserType &ser,
+                                                               GLuint framebufferHandle,
+                                                               GLsizei numAttachments,
+                                                               const GLenum *attachments)
+{
+  SERIALISE_ELEMENT_LOCAL(framebuffer, FramebufferRes(GetCtx(), framebufferHandle));
+  SERIALISE_ELEMENT(numAttachments);
+  SERIALISE_ELEMENT_ARRAY(attachments, numAttachments);
+
+  SERIALISE_CHECK_READ_ERRORS();
+
+  if(IsReplayingAndReading())
+  {
+    if(framebuffer.name == 0)
+      framebuffer.name = m_CurrentDefaultFBO;
+
+    GL.glInvalidateNamedFramebufferData(framebuffer.name, numAttachments, attachments);
+
+    AddResourceInitChunk(framebuffer);
```

Comment:
This isn't an initialisation chunk as it's serialised mid-frame, so we shouldn't call `AddResourceInitChunk` here.